using-kubecost/navigating-the-kubecost-ui/teams.md

```diff
@@ -23,79 +20,249 @@ persistentVolume:
 
-Admins can edit existing teams by selecting them from the Teams page. Teams can then also be deleted by selecting *Delete Team*.
+![Adding a Role](/images/rbac-teams-role-creation-example.png)
 
-## Notes about user roles
+To add a role:
 
-User roles can always be optionally configured using the UI or by adjusting the Helm flags in your *values.yaml*. There is no reequirement to choose one or the other.
+1. Click the `New Role` button on the roles page.
+2. Add a role name. Note that this must be unique, and multiple roles cannot share the same name.
+3. Add a description of the role.
+4. Choose the role's access level. This can be `Admin`, `Read Only`, or `Editor`.
+5. Define which pages can be viewed by users with the role. If any `Monitoring` pages are shown, you may additionally define a filter that will apply to the contents of this page.
 
-Kubecost RBAC follows the principle of most privilege. If a user is added to multiple teams with different roles, they will assume the role with the highest authority unless Allocation Filters are used to limit role functions.
+### Adding a team through the Kubecost UI
 
-Authentication is handled by the cost-model and Aggregator pods.
+![Adding a Team](/images/rbac-teams-team-creation-example.png)
 
-## Debugging
+To add a team:
 
-For help with troubleshooting, follow this guide for diagnostic assistance:
+1. Click the `New Team` button on the teams page.
+2. Add a team name. Note that this must be unique, and multiple teams cannot share the same name.
+3. Define user claims. For more information on what a claim is, see the [Claims](teams.md#claims) section above. __Note that any overlap between a user's claim data from the IdP and a team's claims will assign the user to the team, even if there are other claims specified in the Team.__
+4. Add one or more roles.
+
+On login, the user with the associated claims will be assigned the permissions of their team(s) role(s).
+
+Note that, depending on the claims set, a user may have multiple teams. Also note that a team can have multiple roles. For more info, see the [Multiple teams](teams.md#multiple-teams) section below.
+
+## Multiple teams
+
+If a user has multiple roles (either through having multiple teams, a team with multiple roles, or both), they will be assigned an __additive__ union of the permissions defined by each role.
+
+### Filters
+
+Asset, Allocation, and Cloud Cost filters are ANDed together within a role, and then ORed together within both a set of roles in a team and a set of teams for a user.
+
+For example, if a user is assigned both roles `Role 1` and `Role 2` with the following Allocation filters:
+
+```
+Role 1:
+cluster IS cluster-1, cluster-2
+namespace IS kubecost
+
+Role 2:
+cluster IS cluster-3
+namespace IS dev
+```
+The user will gain a combined filter of `((cluster = cluster-1 OR cluster = cluster-2) AND namespace = kubecost) OR (cluster = cluster-3 AND namespace = dev)`.

```

Comment:
```suggestion
```

The user will gain a combined filter of `((cluster = cluster-1 OR cluster = cluster-2) AND namespace = kubecost) OR (cluster = cluster-3 AND namespace = dev)`.
```